erlang/pop_protocol/pop_protocol.erl

```diff
@@ -124,6 +151,15 @@ add_one_block(Block, CurrentTime, ProtocolData)
 get_genesis_tree_data(CurrentTime) ->
     TD0 = #tree_data{},
     B0 = blocktree:generate_new_block(undefined, TD0),
-    B1 = B0#{consensus_data := #{timestamp => CurrentTime}},
+    B1 = B0#{
```

Comment:
so B0#{...} means a copy of B0 with the following values changed?

Reply:
Ya